storyTwo.php

```diff
@@ -0,0 +1,63 @@
+<?php
+require 'functions.php';
+
+if (isset($_POST['submit'])) {
+    $db = connectToDB();
+    $errors = validate($_POST, $db);
+
+    if (insertNewCar($errors, $_POST, $db)) {
+        header('Location: carsCollection.php');
+        exit;
```

Comment:
I don't think this is doing anything because `header()` already redirects the whole page (I might be wrong) - please remove this line if it isn't necessary.